extensions/audio-device/src/set-output-device.tsx

```diff
@@ -1,6 +1,9 @@
-import { render } from "@raycast/api";
+import { getPreferenceValues } from "@raycast/api";
 import { DeviceList } from "./helpers";
+import { AirPlaySelector } from "./airplay";
 
-export default async function main() {
-  render(<DeviceList type="output" />);
+export default function Command() {
+  const preferences = getPreferenceValues();
+  return preferences.airplay ? <AirPlaySelector /> : <DeviceList type="output" />;
```

Comment:
Instead of having to switch between the 2 with a preference, do you think it'd be possible to load both in the same list? I'd be fine if some (the airplay ones) appear a bit later I think. The question is whether we can avoid duplicates